### ISSUE TITLE
libunwind-1.1.tar.gz sha1 mismatch

### ISSUE THREAD

**Issue by jam7** [skycocker/chromebrew#320]
I've just noticed libunwind-1.1.tar.gz has aa04e6a7164aef754415b3b5e9453985741dd99e sha1 now.
And `crew install libunwind` fails with following error on c100pa (arm):

```
libtool: link: ranlib .libs/libunwind-dwarf-local.a
libtool: link: rm -fr .libs/libunwind-dwarf-local.lax
libtool: link: ( cd ".libs" && rm -f "libunwind-dwarf-local.la" && ln -s "../libunwind-dwarf-local.la" "libunwind-dwarf-local.la" )
depbase=`echo elf32.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/bash ../libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-arm -I. -D_GNU_SOURCE -
DNDEBUG  -g -O2 -fexceptions -Wall -Wsign-compare -MT elf32.lo -MD -MP -MF $depbase.Tpo -c -o elf32.lo elf32.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-arm -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -
Wall -Wsign-compare -MT elf32.lo -MD -MP -MF .deps/elf32.Tpo -c elf32.c  -fPIC -DPIC -o .libs/elf32.o
In file included from elf32.c:3:0:
elfxx.c:33:18: fatal error: lzma.h: No such file or directory
 #include <lzma.h>
                  ^
compilation terminated.
```

It looks like we need to install liblzma since OS has /usr/lib/liblzma.so.

**Comment by jam7** [comment]
I recently bought x86_64 chromebook and it shows identical problem.  I'm having this problem on both x86_64 and armv7l.

**Comment by thedamian** [comment]
I calculate that the correct sha should have been 07ea3c55e67e26fb7a2ab00ecc67c9e80130ad42

I've merged a PR https://github.com/skycocker/chromebrew/pull/358 that should fix this.

Please test.

**Comment by jam7** [comment]
For me, it is still aa04e6a7164aef754415b3b5e9453985741dd99e.  I try https://... URL (changing to use https), but the same result.

**Comment by cstrouse** [comment]
I also calculate the sum as `aa04e6a7164aef754415b3b5e9453985741dd99e`.

**Comment by cstrouse** [comment]
@jam7 I have `xz-utils` installed on my x86_64 machine because I'm I at one point ran dev_install. `lzma` is a symlink to `xz`. I'll create a package for `xz-utils` in an hour or two.

**Comment by jam7** [comment]
@cstrouse thank you for the confirmation and information about xz-utils.  I'll try it once you make a PR.  thanks.

**Comment by cstrouse** [comment]
@jam7 Let me know if #363 works for you.

**Comment by jam7** [comment]
@cstrouse it works fine.  thanks.  only sha1sum is problem now.

**Comment by cstrouse** [comment]
Anyone figure out what the issue with the digest mismatch was?

**Comment by jam7** [comment]
closed in favor of #603
